README.md

```diff
@@ -65,13 +62,16 @@ Constructing your own widgets (Custom widgets) is simple and straightforward.
 The smallest `@dojo/widget-core` example looks like this:
 
 ```ts
-const createMyWidget = createWidgetBase.extend({
-   render(): DNode {
+class MyWidget extends WidgetBase<WidgetProperties> {
+	render() {
        return v('h1', { innerHTML: 'Hello, Dojo!' });
-   }
-});
+	}
+}
 
-createMyWidget.mixin(createProjectorMixin)().append();
+const Projector = ProjectorMixin(MyWidget);
```

Comment:
And yeah the idea is to not keep cluttering examples with imports, but as this is the first one perhaps we should.